esp/services/esdl_svc_engine/esdl_binding.cpp

```diff
@@ -639,7 +639,8 @@ enum EsdlMethodImplType
     EsdlMethodImplWsEcl,
     EsdlMethodImplProxy,
     EsdlMethodImplJava,
-    EsdlMethodImplCpp
+    EsdlMethodImplCpp,
+    EsdlMethodImplScript
```

Comment:
The call to configureUrlMethod (line 618), or its implementation, should be made aware of this type. It records a user warning regarding the now unnecessary url attribute before returning.

Reply:
done.